src/rabbit_amqqueue.erl

```diff
@@ -201,6 +203,11 @@
 -spec(sync_mirrors/1 :: (pid()) -> 'ok' | rabbit_types:error('not_mirrored')).
 -spec(cancel_sync_mirrors/1 :: (pid()) -> 'ok' | {'ok', 'not_syncing'}).
 
+-spec(pid_of/1 :: (rabbit_types:amqqueue()) ->
+  {'ok', pid()} | rabbit_types:error('not_found')).
+-spec(pid_of/2 :: (rabbit_types:vhost(), rabbit_amqqueue:name()) ->
```

Comment:
The `rabbit_amqqueue:name()` type is a `#resource{}` record, not a binary if I understand correctly. `rabbit_misc:r/3` expects a binary.